tests/lib/common.js

```diff
@@ -8,39 +9,39 @@ function testWidgetDefaults( widget, defaults ) {
 	var pluginDefaults = $.ui[ widget ].prototype.options;
 
 	// Ensure that all defaults have the correct value
-	test( "defined defaults", function() {
+	QUnit.test( "defined defaults", function( assert ) {
```

Comment:
Let's make the switch to no globals for `QUnit`/`assert` separately. We can land that in master much sooner than Intern, and will make the diff at the end much smaller and more focused.